tests/v2/base_test.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import os
+from app.api.v2.models.db import init_db
+from app import app
+
+KEY = os.getenv('SECRET_KEY')
+ADMIN_TOKEN = os.getenv("ADMIN_TOKEN")
+
+
+class BaseTestClass(unittest.TestCase):
+    """
+    Setting up tests
+    """
+
+    def setUp(self):
+        self.app = app("testing")
+        self.client = self.app.test_client()
+        self.app.config['TESTING'] = True
```

Comment:
If you set up your config file correctly, you won't need this line

Reply:
Thanks for pointing this out,

In the next commit, I'll make the change and will also be using the same config properties to choose the database to use for either testing